### DISCUSSION TITLE
Filtering google spreadsheet data provider

### DISCUSSION THREAD

**Discussion by diogogcunha** [Budibase/budibase#8556] [Help] [answered]
It seems like the static filtering for the google sheets data provider is not working at all. Is there any limitations to it?

![image](https://user-images.githubusercontent.com/115067/200312801-a16603ef-9763-447f-9f74-c1cd16b28aad.png)

**Comment by melohagan** [comment, marked as answer]
Hey @diogogcunha 

There is a limitation at the moment. Duplicate here: https://github.com/Budibase/budibase/discussions/8536
